src/langcheck/augment/en/_gender/_gender_pronouns.py

```diff
@@ -0,0 +1,137 @@
+
+_PLURAL_PRONOUNS = _BaseGenderPronouns(
+    subject='they',
+    dependent_possessive='their',
+    object='them',
+    independent_possessive='theirs',
+    reflexive='themselves',
+)
+
+# Neutral pronouns list were taken from some of the most popular neutral
+# pronouns from the following sources:
+# https://ofm.wa.gov/sites/default/files/public/shr/Diversity/BRGs/Pronouns%20Compilation%20Best%20Practices.pdf
+# https://uwm.edu/lgbtrc/support/gender-pronouns/
+_NEO_PRONOUNS_LIST = [
+    _BaseGenderPronouns(
+        subject='zie',
+        dependent_possessive='zir',
+        object='zim',
+        independent_possessive='zis',
+        reflexive='zieself',
+    ),
+    _BaseGenderPronouns(
+        subject='zie',
+        dependent_possessive='hir',
+        object='hir',
+        independent_possessive='hirs',
+        reflexive='hirself',
+    ),
+    _BaseGenderPronouns(
+        subject='ze',
+        dependent_possessive='hir',
+        object='hir',
+        independent_possessive='hirs',
+        reflexive='hirself',
+    ),
+    _BaseGenderPronouns(
+        subject='ze',
+        dependent_possessive='zir',
+        object='zim',
+        independent_possessive='zis',
+        reflexive='zieself',
+    ),
+    _BaseGenderPronouns(
+        subject='sie',
+        dependent_possessive='hir',
+        object='sie',
+        independent_possessive='hirs',
+        reflexive='hirself',
+    ),
+    _BaseGenderPronouns(
+        subject='ey',
+        dependent_possessive='eir',
+        object='em',
+        independent_possessive='eirs',
+        reflexive='eirself',
+    ),
+    _BaseGenderPronouns(
+        subject='ve',
+        dependent_possessive='vis',
+        object='ver',
+        independent_possessive='vers',
+        reflexive='verself',
+    ),
+    _BaseGenderPronouns(
+        subject='tey',
+        dependent_possessive='ter',
+        object='tem',
+        independent_possessive='ters',
+        reflexive='terself',
+    ),
+    _BaseGenderPronouns(
+        subject='e',
+        dependent_possessive='eir',
+        object='em',
+        independent_possessive='eirs',
+        reflexive='ems',
+    ),
+    _BaseGenderPronouns(
+        subject='per',
+        dependent_possessive='per',
+        object='per',
+        independent_possessive='pers',
+        reflexive='perself',
+    ),
+    _BaseGenderPronouns(
+        subject='xe',
+        dependent_possessive='xyr',
+        object='xem',
+        independent_possessive='xyrs',
+        reflexive='xyrself',
+    ),
+    _BaseGenderPronouns(
+        subject='fae',
+        dependent_possessive='faer',
+        object='faer',
+        independent_possessive='faers',
+        reflexive='faerself',
+    ),
+    _BaseGenderPronouns(
+        subject='ae',
+        dependent_possessive='aer',
+        object='aer',
+        independent_possessive='aers',
+        reflexive='aerself',
+    )
+]
+
+_PRONOUNS_DICT = MappingProxyType({
+    None: [_PLURAL_PRONOUNS],
```

Comment:
From @kennysong : Could be better to name it "plural".

Reply:
Updated.